teradatalabs/mapr52-hive/files/wardenTracker.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+hname=$(hostname)
+
+Services=0
+
+maprcliReady=$(maprcli service list -node $hname | grep 'ERROR (10009)' | wc -l)
+
+# WAIT FOR CLDB TO START
+while [ $Services -ne 2 ]
+do
+if [ $maprcliReady == 1 ]
+then
+   maprcliReady=$(maprcli service list -node $hname | grep 'ERROR (10009)' | wc -l)
+   Services=0
+else
+   Services=$(maprcli service list -node $hname | grep CLDB |awk '{$1=$1};1' | tr ' ' '\n' | tail -1f)
```

Comment:
dockerize will not work in this case as in mapr port opens very late and there is no finite time to open port @ArturGajowy  , similar issue we faced in the test container also and i was forced to remove expose_hive and create new function expose_mapr